src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -37,7 +38,7 @@ public ResponseEntity createLine(@RequestBody LineRequest lineRequest) {
     }
 
     @PutMapping("/{id}")
-    public ResponseEntity updateLine(@PathVariable Long id, @RequestBody LineRequest lineUpdateRequest) {
+    public ResponseEntity updateLine(@PathVariable Long id, @RequestBody LineUpdateRequest lineUpdateRequest) {
```

Comment:
ResponseEntity를 사용할때는 제네릭 타입을 명시해주는 것이 좋습니다 :)
```suggestion
    public ResponseEntity<Void> updateLine(@PathVariable Long id, @RequestBody LineUpdateRequest lineUpdateRequest) {
```